ansible/roles/runner/tasks/setup.yml

```diff
@@ -46,6 +47,9 @@
   notify:
     - restart_nfs
 
+- name: installing libvirt plugin for vagrant
+  shell: vagrant plugin install vagrant-libvirt
```

Comment:
If you want to install the vagrant-libvirt plugin in this way, it'd probably be best if you'd remove the `vagrant-libvirt` from package installation above (line 22) to avoid any confusion.